.github/workflows/ctest_driver_script.cmake

```diff
@@ -2,7 +2,7 @@ set(CTEST_SITE "${SITENAME}")
 
 cmake_host_system_information(RESULT DISTRIB_ID QUERY DISTRIB_ID)
 cmake_host_system_information(RESULT DISTRIB_VERSION_ID QUERY DISTRIB_VERSION_ID)
-set(CTEST_BUILD_NAME "${DISTRIB_ID}-${DISTRIB_VERSION_ID}-${DATABASE}-${STORAGE_TYPE}")
+set(CTEST_BUILD_NAME "${DISTRIB_ID}-${DISTRIB_VERSION_ID}-${STORAGE_TYPE}")
```

Comment:
I left this and related code deliberately with the idea that it would be beneficial to run CI against multiple Postgres versions in the future, but I've held off on doing so since the tests have been flaky.  No objections to removing it and re-adding later.